packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -32,18 +33,37 @@ export interface CommerceApiProviderProps extends ApiClientConfigParams {
  */
 export const CommerceApiContext = React.createContext({} as ApiClients)
 
+interface configType {
+    proxy: string
+    headers: {
+        authorization?: string
+    }
+    parameters: {
+        clientId: string
+        organizationId: string
+        shortCode: string
+        siteId: string
+    }
+    throwOnBadResponse: boolean
+}
 /**
  * Initialize a set of Commerce API clients and make it available to all of descendant components
  *
  * @param props
  * @returns Provider to wrap your app with
  */
+const localhost = 'http://localhost:3000'
+// Input your token here to for the apiClients to work
+// this is a temporary solution to implement hooks while waiting for slas hook
+const token = ''
 const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
-    const {children, clientId, organizationId, shortCode, siteId, proxy} = props
-
-    const config = {
-        proxy,
-        headers: {},
+    const {children, clientId, organizationId, shortCode, siteId} = props
```

Comment:
Please do not pay attention to this code change here, I made some change to grab the token from slas so I can see the hooks in action.